src/box/lua/load_cfg.lua

```diff
@@ -423,6 +399,17 @@ local dynamic_cfg_modules = {
             flightrec_requests_max_res_size = true,
         },
     },
+    log = {
+        cfg = log.internal.box_api_cfg,
+        settings = log.cfg,
+        skip_at_load = true,
```

Comment:
There's a bug somewhere in this code. The following commands work although they shouldn't:

```
tarantool> require('log').cfg{log = 'foo.log'}
---
...

tarantool> box.cfg{log = 'bar.log'}
---
...
```

Please fix and add a test case.

Reply:
Bug is fixed and test is added.